SimMuon/MCTruth/python/NewMuonAssociatorByHits_cfi.py

```diff
@@ -0,0 +1,147 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
"New" should not be used to name your classes. instead use something descriptive.